wscript

```diff
@@ -309,9 +309,11 @@ def _write_version_header(tsk):
 
 
 def build(bld):
-    config_hash = Utils.h_file(bld.bldnode.make_node('ap_config.h').abspath())
-    bld.env.CCDEPS = config_hash
-    bld.env.CXXDEPS = config_hash
+    config_node = bld.bldnode.make_node('ap_config.h').abspath()
+    if os.path.exists(config_node):
+        config_hash = Utils.h_file(config_node)
+        bld.env.CCDEPS = config_hash
+        bld.env.CXXDEPS = config_hash
```

Comment:
The header `ap_config.h` is generated during configuration. If the build isn't configured, Waf raises an error before getting here. I can't really see the problem this commit addressing. Could you provide an example of reproducing the error?